### ISSUE TITLE
Merge SuiteName into CCTBX

### ISSUE THREAD

**Issue by russell-taylor** [ReliaSolve/Molprobity2#123]
When @Joymaker says that it is ready to test, remove suitename module builds and add the subdirectory into the build tree.  Then test building and running molprobity.suitename.  Then push into a native-suitename branch of the cctbx organization and let Michael and others try it out.  Once it is shown to to work, announce and then merge into master.

**Comment by russell-taylor** [comment]
(done) The new report format is slightly different

Old format, no space before ==:

For all 0 suites: average suiteness== 0.000 (power==3.00)

New format, space before == and missing power clause:

For all 65 suites: average suiteness == 0.658

**Comment by russell-taylor** [comment]
(done) The new version doesn't support the MolProbity workflow

`phenix.suitename -report -pointIDfields 7 -altIDfield 6 < infile`

Errors:
Sorry: Some PHIL parameters are not recognized by suitename.
 Please run this program with the --show-defaults option to see what parameters
are available.
 PHIL parameters in files should be fully specified (e.g. "output.overwrite"
instead of just "overwrite")

**Comment by russell-taylor** [comment]
(done) Do the pull request into master on cctbx and let Billy know to run his full suite of tests before merging.

**Comment by russell-taylor** [comment]
Waiting on Billy to merge
